cmd/cosign/cli/verify_dockerfile.go

```diff
@@ -122,9 +122,10 @@ func getImagesFromDockerfile(dockerfile io.Reader) ([]string, error) {
 }
 
 func getImageFromLine(line string) string {
-	line = strings.TrimPrefix(line, "FROM") // Remove "FROM" prefix
-	line = os.ExpandEnv(line)               // Substitute templated vars
-	line = strings.Split(line, " AS ")[0]   // Remove the "AS" portion of line
+	line = strings.TrimPrefix(line, "FROM")     // Remove "FROM" prefix
+	line = os.ExpandEnv(line)                   // Substitute templated vars
+	line = strings.ReplaceAll(line, "as", "AS") // replace lower "as" with "AS"
```

Comment:
this would substitute all instances of "as". I'm not sure if image names are case sensitive, but URL paths can be